ttle.py

```diff
@@ -101,10 +106,48 @@ def __init__(self):
         ShowBase.__init__(self)
         aspect2d.setAntialias(AntialiasAttrib.MAuto)
 
+        # Create the framerate meter
+        flag = self.config.GetBool('show-frame-rate-meter', False)
+        if flag:
+            self.toggleFrameRateMeter(flag)
+
         from toontown.leveleditor import LevelEditor
         self.le = LevelEditor.LevelEditor()
         self.le.startUp(args.dnaPath)
 
+    def setFrameRateMeter(self, flag):
+        return
+
+    def toggleFrameRateMeter(self, flag):
+        if flag:
+            if not self.frameRateMeter:
+                self.frameRateMeter = OnscreenText(parent=base.a2dTopRight, text='', pos=(-0.1125, -0.05, 0.0), scale=0.05, bg=(0, 0, 0, 0.4), font=ToontownGlobals.getToonFont())
```

Comment:
add `style = 3` here for consistency, also spaces between the `=` is the code style of this program